### ISSUE TITLE
Quick Draft from dashboard does not work

### ISSUE THREAD

**Issue by jcklpe** [WordPress/gutenberg#12404]
**Describe the bug**
I create a quick draft from the dashboard of a live production site and while it saves the blog post draft it does not save the contents of the body

**To Reproduce**
1. Go to dashboard
2. Click on and enter quick draft content
3. Submit
4. Go to blog and open draft. Draft has empty body paragraph

**Expected behavior**
I expect the body paragraph information that I entered in the quick draft widget to be there. It isn't. 

**Screenshots**
![image](https://user-images.githubusercontent.com/9451726/49188038-fc324c00-f32e-11e8-925c-bbaf0c281373.png)

Weirdly enough this test post shows up?

![image](https://user-images.githubusercontent.com/9451726/49188172-63500080-f32f-11e8-8f3e-9bbb16b27f6e.png)

But if you look at the other posts (which one of them has it's intended contents still visible from the first screenshot, you'll see that it's broken. 

![image](https://user-images.githubusercontent.com/9451726/49188219-87134680-f32f-11e8-9ee5-a170a15ed3a8.png)

The embedded link was added after opening up the full edit to add an additional link and finding that it ran into this problem. 

I'm running Wordpress 5.0 with gutenberg plugin activated (because I want the latest fixes on the editor since it's stills pretty buggy)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox
 - Version latest stable

**Comment by jcklpe** [comment]
Going through all the drafts I generated this way I think I see the common thread. The ones that worked are the ones that were just text. Anything with a link broke, probably because the quick draft tries to put everything in a classic block and G-berg is probably trying to force the link into some kind of embded block. 

Also as an aside, I find that embedded link behavior REALLY annoying when I'm trying to write. Sometimes I literally just want a link. and I don't want to have the stupid thing just break my entire paragraph block off because it wants to try and embed (which like 90% of the time it doesn't even do the embed right)

**Comment by mrwweb** [comment]
Just tested this on Firefox 63.0.3, WordPress 5.0-RC1-43946, no plugins, Twenty Nineteen

Both testing with and without a link in quick draft worked for me, including posts that contained a link and text and posts that only contained a link or multiple links.

A big +1 to not hijacking any naked URL and trying to turn it into an embed. That's something I frequently do when drafting content and saving a URL for use later.

**Comment by designsimply** [comment]
I tested three different ways:

* WordPress 4.9.8 and Gutenberg 4.5.1
* WordPress 5.0-RC1-43946
* WordPress 5.0-RC1-43946 and Gutenberg 4.5.1

I used the following sample content to create the following four Quick Drafts: 

Title: Quick Draft

> This post was made as a Quick Draft.

Title: boygenius

> https://www.youtube.com/watch?v=OS48Lp34Zic
> The group is new, but all of the members of boygenius — Julien Baker, Lucy Dacus and Phoebe Bridgers — are Tiny Desk Concert alumae.

Title: Alaska by Maggie Rogers

> https://www.youtube.com/watch?v=SqPtIkxSxI0
> 
> "Alaska" — a song whose buoyantly clever arrangement translates beautifully to the solo acoustic treatment she gives it here.

Title: A Myth of Two Souls

> https://thephotographersgallery.org.uk/whats-on/mario-klingemannn-neurography

Results: the content 

1. Text only came through.
1. YouTube URL was not on its own line and did not get turned into a link or embed.
1. YouTube URL on it's own line was turned into an embed.
1. URL that is not an embed was not turned into an embed.

![screen shot 2018-11-29 at 8 28 02 am](https://user-images.githubusercontent.com/1119271/49232210-c0d86180-f3b0-11e8-9a76-117d01c406b4.png)
<sup>Seen at https://madefortesting.com/wp-admin/post.php?post=743&action=edit running WordPress 5.0-RC1-43946 and Gutenberg 4.5.1 using Firefox 63.0.3 on macOS 10.13.6.</sup>

> I'm running Wordpress 5.0 with gutenberg plugin activated (because I want the latest fixes on the editor since it's stills pretty buggy)

The versions were a little less out of sync for early betas but that was fixed and the plugin and 5.0-RC1 are lining up extremely closely and so this shouldn't be necessary right now. If you did want the very latest fixes, you would want to be testing with the `master` version of the plugin (but again, it won't be very different at this point in time!).

@jcklpe I tested with what I think is similar sample content to what you described and what I see in your screenshots. If you're still having trouble, maybe could you include the sample content that's breaking for you in a reply for testing purposes?

**Comment by jcklpe** [comment]
@designsimply I was able to reproduce the issue a second time with a fresh quick draft. 

Quick draft:
![image](https://user-images.githubusercontent.com/9451726/49240255-63e9a500-f3ca-11e8-8158-bcaefc2e571f.png)


End result:
![image](https://user-images.githubusercontent.com/9451726/49240169-36046080-f3ca-11e8-955f-d532a5916689.png)

I made sure to try multiple links but even a single link breaks it. 

Also in case it helps here's a read out of my active plugin list from wp-cli, though I don't think I have anything installed that's very unusual. 

```
+--------------------------------+--------+--------+-------------+
| name                           | status | update | version     |
+--------------------------------+--------+--------+-------------+
| codepress-admin-columns        | active | none   | 3.3.1       |
| admin-menu-editor              | active | none   | 1.8.5       |
| advanced-custom-fields-pro     | active | none   | 5.8.0-beta3 |
| custom-post-type-ui            | active | none   | 1.6.0       |
| enable-media-replace           | active | none   | 3.2.7       |
| gutenberg                      | active | none   | 4.5.1       |
| insert-pages                   | active | none   | 3.4.3       |
| medium                         | active | none   | 1.4.0       |
| phoenix-media-rename           | active | none   | 1.3.1       |
| physical-custom-upload-folder  | active | none   | 1.0.4       |
| post-types-order               | active | none   | 1.9.3.9     |
| regenerate-thumbnails-advanced | active | none   | 1.6.3.2     |
| wordpress-beta-tester          | active | none   | 1.2.6       |
| wp-cerber                      | active | none   | 7.9         |
| wp-code-highlightjs            | active | none   | 0.6.2       |
| real-media-library             | active | none   | 4.0.8       |
| wp-rocket                      | active | none   | 3.2.2       |
| wp-sync-db                     | active | none   | 1.6.0       |
| wordpress-seo                  | active | none   | 9.2.1       |
+--------------------------------+--------+--------+-------------+
```

**Comment by jcklpe** [comment]
Ah, so to get that plugin list I had to install wp-cli on my new host (just switched to vps and I've been putting off doing it). 

And since I had wp-cli installed I decided that I should disable the beta tester plugin so that I could manually update to release clients via CLI whenever I wanted to (since I'd only really installed the plugin to get on 5.0RC and I wasn't planning on rolling a bleeding edge beta for my live production website.)

And since I was at it, I also deactivated the Gutenberg plugin since ya'll said that was basically the same. And then I thought... hmmm... maybe I should test again. 

And it works now!

![image](https://user-images.githubusercontent.com/9451726/49241034-4fa6a780-f3cc-11e8-8980-4c53035dbdb7.png)


I turned back on g-berg plugin, just to narrow the issue down and it broke again! 

So it seems that it has to do with when g-berg is installed and perhaps specifically when it's installed over WP 5.0 RC. 

Which... well seems like a pretty narrow time period for this to be an issue? Maybe it's something ya'll should check out more, but please feel free to close this issue if ya'll consider this basically taken care of.

**Comment by designsimply** [comment]
Good to know about 5.0+plugin—though I did try testing that scenario too and my test was okay! 

To clarify, it's breaking for you with WordPress 5.0-RC1-43946 + Gutenberg 4.6 and all of the plugins noted in https://github.com/WordPress/gutenberg/issues/12404#issuecomment-442925763 active?

Any chance you are able to temporarily deactivate all of the other plugins except Gutenberg (or maybe just a few like advanced-custom-fields-pro, insert-pages, medium, wp-code-highlightjs, wordpress-seo) as a test to try to rule out a plugin+plugin conflict?

**Comment by jcklpe** [comment]
Yes it is as you said, wordpress 5.0 plus all the plugins listed in the list. 

Unfortunately I've uninstalled the Beta and Gutenberg plugins and it's a live production website that I'm currently using the look for my next job so I am hesitant to test it too much. And like I said it seems like a pretty narrowly applicable issue that will basically fix itself once WP 5.0 officially comes out and Gberg as a separate plugin is retired.

**Comment by designsimply** [comment]
Looping back to check this again now that 5.0 has launched! 

1. Go to dashboard
1. Click on and enter quick draft content
1. Submit
1. Go to blog, open drafts and check to see whether the draft has an empty body paragraph

Result: draft works normally, both links are present, the YouTube link on it's own line turns into an embed as expected.

Title: testing

> https://www.youtube.com/watch?v=a36LAo4YxfA
> 
> https://en.wikipedia.org/wiki/Cybernetics

![12404](https://user-images.githubusercontent.com/1119271/50033649-a27c8380-ffb6-11e8-83b5-b78270b68c1b.gif)

Closing since I couldn't see the problem happen in a new test after the software has been updated and because you mentioned, "please feel free to close this issue if ya'll consider this basically taken care of." If the problem crops up again, please re-open!
